tests.toml

```diff
@@ -0,0 +1,21 @@
+test_format = 1.0
+
+[default]
+
+    # ------------
+    # Tests to run
+    # ------------
+
+    exclude ="install.nourl"
```

Comment:
This shouldn't be needed because it's a webapp so that test won't be run by default (also be careful that `exclude` is expected to be a list, not a single string ;P)